benches/main/line.rs

```diff
@@ -36,4 +36,4 @@ fn line_render(criterion: &mut Criterion) {
 }
 
 criterion_group!(benches, line_render);
-criterion_main!(benches);
+// criterion_main!(benches);
```

Comment:
Why not delete these? 💭

Reply:
I did locally - just missed committing that :)

Reply:
a20d8aca4...4384ca479